model/Ontology/taosystemstatus.rdf

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0"?>
+<rdf:RDF 
+	xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" 
+	xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" 
+	xml:base="http://www.tao.lu/Ontologies/TAOTestCenter.rdf#"
```

Comment:
not clear how have you managed this RDF,
I guess you need to create `xml:base="http://www.tao.lu/Ontologies/TAOSystemStatus.rdf#"`
and then
`http://www.tao.lu/Ontologies/TAOSystemStatus.rdf#taoSystemStatusManager`